pro/src/components/NewNavReview/NewNavReviewDialog/NewNavReviewDialog.tsx

```diff
@@ -79,7 +79,7 @@ const NewNavReviewDialog = ({
             <h1 className={styles['dialog-title']}>Votre avis compte !</h1>
             <fieldset>
               <legend className={styles['radio-legend']}>
-                Selon vous, cette nouvelle interface est ... ? *
+                Selon vous, cette nouvelle interface est ... ?
```

Comment:
[détail] Au passage, est-ce qu'on ne devrait pas corriger les points de suspension et retirer l'espace en trop le précédant ?

```suggestion
                Selon vous, cette nouvelle interface est… ?
```